avm/ptn/deployment-script/import-image-to-acr/CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+The latest version of the changelog can be found [here](/Azure/bicep-registry-modules/blob/main/avm/ptn/deployment-script/import-image-to-acr/CHANGELOG.md).
+
+## 0.4.4
+
+### Changes
+
+- Updated resource provider versions
+- In the test cases, the Azure Container Registry, storage account and Key Vault names are now unique to prevent deployment failures due to duplicate names
+- Updated az cli version in the deployment script
+- Added min and max length to the ACR name property
+- Exclude Content Trues psrule `Azure.ACR.ContentTrust` for this module
+- Disabled contentTrust in the 'max' and 'waf-aligned' tests, as creating signed images is a big overhead for the CI deployment testing and raise an error in the PSRule `Azure.ACR.ContentTrust`
```

Comment:
I guess this is not the case anymore. Anyway I would not add psrule exclusions in the changelog as that is a CI specific change, not part of the module and may be confusing for module consumers, especially if they don't know what PSRule is

```suggestion
```